### DISCUSSION TITLE
Future Idea: Touch interaction

### DISCUSSION THREAD

**Discussion by VentuzDanielWiller** [VentuzTechnology/Ventuz.Extension.prerelease#19] [Ideas]
access to our Touch System

**Comment by gielians** [comment]
Maybe there is a way to determine which face or mesh has been touched maybe even with coordinates to include different touch behaviours  for different areas

**Reply by VentuzRalfStanke** [reply]
We do not know yet, how the touch integration will be. Ventuz has multiple options to perform the "hit-test" where also a face-based version is available (but slow). We will see. Thanks, for the idea!
